projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -809,7 +809,7 @@ function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
 
 	// Placeholder experiment key, we need to update this to the new experiment key once it's created.
 	$experiment_assignment              = \ExPlat\assign_given_user( 'calypso_plans_global_styles_personal_20240127', $owner );
-	$has_global_styles_in_personal_plan = 'treatment' === $experiment_assignment;
+	$has_global_styles_in_personal_plan = $experiment_assignment && 'control' !== $experiment_assignment;
```

Comment:
I think the control variation is serialized as `null`. See PCYsg-Fq9-p2

> With an A/B experiment, participants are assigned to a `control` (returns as `null`) or `treatment` group.

Is this not the case for A/B/C tests?

Reply:
I've updated the condition.